### PR TITLE
Document LanguageClient

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

**Comment by Razzeee** [comment]
Can you also amend this table?

**Comment by lpil** [comment]
I'm afraid I don't know the all answers. I could put question marks in the ones I'm unsure about?

**Comment by Razzeee** [comment]
Yes, better then nothing :)

**Comment by lpil** [comment]
Updated :)

File: README.md [hunk 1]
```diff
@@ -78,12 +78,13 @@ Settings may need a restart to be applied.
...
+| [VSCode](https://github.com/elm-tooling/elm-language-server#vscode)     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| [VIM CoC](https://github.com/elm-tooling/elm-language-server#cocnvim)   | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| [VIM LanguageClient](https://github.com/autozimu/LanguageClient-neovim) | :heavy_check_mark: | :heavy_check_mark: | :grey_question:    | :heavy_check_mark: | :grey_question:    | :grey_question:    | :grey_question:    | :grey_question:    | :grey_question:    | :grey_question:    | :grey_question:    |
```
Comment (Razzeee) [inline README.md]: Can you change the link to `https://github.com/elm-tooling/elm-language-server#anguageClient`?

**Comment by Razzeee** [comment]
Thank you!

File: README.md (hunk 1 above)
Comment (Razzeee) [reply README.md]: I made this change after merging :)

**Pull request merged**